README.md

```diff
@@ -14,61 +14,73 @@ with **simple steps** people can **_easily_ follow** <br />
+***Please*** ***inform us*** if ***anything*** is ***unclear***
+or if you have _**a suggestion** for how to **simplify/streamline**_ it!
+[![Get Help!](https://img.shields.io/badge/get-help-brightgreen.svg?style=flat)](https://github.com/dwyl/contributing/issues)
 
-### _Got_ Issues...?
 
-_Everything_ starts with a _thought_.
-We collect our _thoughts_ in GitHub "issues".
-The issue can then be _**dicussed, validated** and **worked on** as a **team**_.
-We use _issues_ to log our user stories and sub/technical tasks
-to ensure we have a
-[single version of the truth](https://github.com/dwyl/github-reference#why).  
+# Part 1: _Describe_ the Idea or User Story in an Issue
+
+### _Why_ Use GitHub Issues...?
 
-_**User stories** describe features_ (_or expected behaviours_)
-of the application in terms of their ***value***
-to the ***person using the application***.
+_Everything_ starts with a _thought_.
+We collect _all_ our _thoughts_ in GitHub "issues". <br />
+The issue can then be _**dicussed, validated**
+and **worked on** as a **team**_ while keeping ***everyone informed***. <br />
+We use _issues_ to _log_ our user stories, sub/technical tasks
+and any ***progress*** towards solving the issue, <br />
+this ***ensures*** we have a
+[_**single source** of the **truth**_](https://github.com/dwyl/github-reference#why)
+that _everyone_ can easily see!  
 
 
 ### A) Check if the Idea/Story/Issue _Already Exists_ Using GitHub Search ![search](https://cloud.githubusercontent.com/assets/194400/22224334/255930a8-e1b6-11e6-9953-12cd4739aa1c.png)
 
-1. Please take a minute to ***read through the existing stories/issues***
-for the project to familiarize yourself with the current "backlog".
+1. Please take a moment to ***read through the existing stories/issues***
+for the project <br />
+to familiarize yourself with the _current_ "***backlog***".
+(_there is a chance what you are thinking has **already** been **described**
+  by someone else..._)
 
 1. Try ***searching*** through the ***existing issues***
 for _keywords_ associated with the story/issue you want to create, <br />
 e.g:
 https://github.com/dwyl/time/issues?utf8=%E2%9C%93&q=is%3Aissue%20app+icon <br/>
-if `"app icon"` were your keywords and you're searching in the "**Time**" project.
-
+if `"app icon"` were the keywords for what you have in mind
+and you were searching in the "**Time**" project. <br />
 ![issue-matching-keyword-search](https://cloud.githubusercontent.com/assets/194400/22225715/852d9f78-e1bb-11e6-9d20-37492da92dc1.png)
+
 1. if you ***find*** an _existing_ issue that matches what you were looking for,
 _please_ click on the title of the issue, ***read the description*** to
 confirm if it's what you are looking for, then ***comment*** on it
-so we know more than one person has the need/issue.
-
+so we know more than one person has the need/issue. <br />
+> _**Note**: if there is **already an issue** for what you were thinking,
+you can **skip** the "B)" and "C)" steps
```

Comment:
@iteles also happy for someone `else` to re-do this but _doubt_ that anyone _will_. 
so if you _want_ the change, please `describe` in an issue so that I can "_fix_" it tomorrow. thanks.